api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java

```diff
@@ -712,6 +722,7 @@ public void testGetUserNotInWhitelistedByState() throws IOException, Interrupted
                 ADMIN_USERNAME, ADMIN_PASSWORD));
     }
 
+    @Ignore // TODO: Renable once #119 is merged
     @Test
     public void testGetUserNotInWhitelistedByLanguageLocationCode() throws IOException, InterruptedException {
```

Comment:
Yep, as long as CI passes

-----Original Message-----
From: "Rob LaRubbio" <notifications@github.com>
Sent: ‎5/‎12/‎2015 4:21 PM
To: "motech-implementations/mim" <mim@noreply.github.com>
Cc: "Lauren Lavoie" <llavoie@grameenfoundation.org>
Subject: Re: [mim] Fixes #147 (#239)

In api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java:
> @@ -712,6 +722,7 @@ public void testGetUserNotInWhitelistedByState() throws IOException, Interrupted
>                  ADMIN_USERNAME, ADMIN_PASSWORD));
>      }
>  
> +    @Ignore // TODO: Renable once #119 is merged
>      @Test
>      public void testGetUserNotInWhitelistedByLanguageLocationCode() throws IOException, InterruptedException {
I updated this test and the other. ok to merge?
—
Reply to this email directly or view it on GitHub.